### DISCUSSION TITLE
过期：懒人包 2022.0903

### DISCUSSION THREAD

**Discussion by hooke007** [hooke007/mpv_PlayKit#171] [回收]
## 版本专帖

下版懒人包将有大幅改动 https://github.com/hooke007/MPV_lazy/issues/168

上游无更新，当期无简易包更新（沿用上期）。

<hr /><em>link <a href='https://github.com/hooke007/MPV_lazy/releases/tag/20220903'>mpv-lazy-20220903</a></em>
<hr /><em>link <a href='https://github.com/hooke007/mpv.net_CM/releases/tag/6031'>mpvnet-CM-20220730</a></em>

**Comment by OmiChrome** [comment]
whoa whoa . :OOOO 

man thsi is hilariousy sick 


omg omg 

that is soooooooooooooooooo gooood . omg

**Comment by OmiChrome** [comment]
btw is there any way so i can import that looks and right-click to normal player mpv ?

**Reply by hooke007** [reply]
This(mpv-lazy) is normal mpv too.
OSC: https://github.com/hooke007/MPV_lazy/discussions/18
context_menu: https://github.com/hooke007/MPV_lazy/discussions/60 It's my fork of this script https://gitlab.com/carmanaught/mpvcontextmenu
It's not recommmand to use my fork because I did lots of Localised modification. Try upstream's script would be better.

**Reply by OmiChrome** [reply]
noice :D

**Comment by skiqing** [comment]
今天突然显示（vapoursynth.Error: RIFE: gpu_thread must be between 1 and 1 (inclusive)），gpu_thread改成1能正常使用，2就不行。

**Reply by hooke007** [reply]
这你得向对应的项目报告，不是mpv的问题

**Reply by skiqing** [reply]
重装AMD驱动正常了，fuck amd

**Comment by SuperBlackWednesday** [comment]
potpalyer 有个功能 就是无论声道是2.0还是7.1 都能强制输出5.1 MPV能实现这个功能吗？

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#options-audio-channels

**Comment by SuperBlackWednesday** [comment]
还有一个问题 我着色器用下面的设置方法提升很明显
~~/shaders/Anime4K_Restore_CNN_VL.glsl;~~/shaders/Anime4K_Clamp_Highlights.glsl;~~/shaders/Anime4K_Upscale_CNN_x2_VL.glsl;~~/shaders/Anime4K_Restore_CNN_M.glsl;~~/shaders/Anime4K_AutoDownscalePre_x2.glsl;~~/shaders/Anime4K_AutoDownscalePre_x4.glsl;~~/shaders/Anime4K_Upscale_CNN_x2_M.glsl
但是显卡飙到90度 噪音特别大  有没有其他的设置方法能提升明显的  交流一下啊

**Reply by hooke007** [reply]
默认你已读过文档，显卡性能不足请减配
